### DISCUSSION TITLE
Import/Export all data from application.

### DISCUSSION THREAD

**Discussion by sabikrahat** [pocketbase/pocketbase#2845] [Q&A]
Hello!,

Is there any way to export all data of each collection in a json file from the application or import all data from a json file directly from the application? I'm using Pocketbase in my Flutter project.

**Comment by ganigeorgiev** [comment]
Other than making individual `getFullList()` requests for the export, there is no built in functionality in PocketBase for export/import of data yet. This eventually will be handled as part of #48 but before that we'll need bulk/transactional web APIs support (_they are planned for v0.18/v0.19_).

**Reply by sabikrahat** [reply]
@ganigeorgiev bro, you have done a very good job and you've planned the whole project very nicely. No problem. I'll wait for the v0.18/0.19.
Thank you so much.
